dftfe2.rc

```diff
@@ -59,46 +82,17 @@ fn install_spglib {
   cd $WD
 }
 
-fn install_p4est {
+fn install_p4est{
   cd $WD/src
-  CFLAGS_FAST='-O2 -fopenmp'
-  CFLAGS_DEBUG='-O0 -g -fopenmp'
-  if(test ! -d p4est) {
-    git clone --recurse-submodules https://github.com/cburstedde/p4est.git
-    bash -c 'cd p4est && git checkout v2.2 && module load libtool && ./bootstrap'
-  }
+  rm -rf p4est
+  mkdir p4est
   cd p4est
-  if(test ! -f src/p4est.h) {
-    echo 'Main header file missing'
-    return 1
-  }
-  mkdir -p FAST
-  cd FAST
-  ../configure --enable-mpi --enable-shared \
-        --disable-vtk-binary --without-blas '--enable-openmp=-fopenmp' \
-        '--prefix='$INST 'CFLAGS='$CFLAGS_FAST \
-        'CC=cc' 'CXX=CC' \
-        'CPPFLAGS=-DSC_LOG_PRIORITY=SC_LP_ESSENTIAL' \
-        > config.output
-  make -C sc -j 8 > make.output
-  make -j 8 >> make.output
-  make install >> make.output
-  echo FAST version installed in $INST
-  cd ..
-
-  # skipping debug build
-  #mkdir -p DEBUG
-  #cd DEBUG
-  #../configure --enable-debug --enable-mpi --enable-shared \
-  #        --disable-vtk-binary --without-blas '--enable-openmp=-fopenmp' \
-  #        '--prefix='$INSTALL_DEBUG 'CFLAGS='$CFLAGS_DEBUG \
-  #        'CPPFLAGS=-DSC_LOG_PRIORITY=SC_LP_ESSENTIAL' \
-  #        > config.output
-  #make -C sc -j 8 > make.output
-  #make -j 8 >> make.output
-  #make install >> make.output
-  #echo "DEBUG version installed in $INSTALL_DEBUG"
-}
+  wget 'https://github.com/p4est/p4est.github.io/raw/master/release/p4est-2.2.tar.gz'
+  wget 'https://raw.githubusercontent.com/dftfeDevelopers/dftfe/manual/p4est-setup.sh'
```

Comment:
This method for setup can introduce errors because the exact version of p4est-setup.sh that is run here is not fixed.  It's much better to inline the setup steps and call out a specific p4est version here directly.